lib/ex_machina/sequence.ex

```diff
@@ -3,6 +3,12 @@ defmodule ExMachina.Sequence do
     Agent.start_link(fn -> HashDict.new end, name: __MODULE__)
   end
 
+  def next(sequence_name) do
```

Comment:
I think we should raise. I think anything else would be unexpected and unlikely to be helpful (if not harmful).